project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.8.2
```

Comment:
I think there was a reason this hasn't been updated, but I don't remember what it was. @armanbilge WDYT?